docs/references/observability.rst

```diff
@@ -362,6 +362,27 @@ For example, to only allow requests from an IP address to get the execution plan
  -- set this function on your postgrest.conf
  -- db-pre-request = filter_plan_requests
 
+.. _content-length_header:
```

Comment:
@laurenceisla Just checked RTD and this should go before https://postgrest--4018.org.readthedocs.build/en/4018/references/observability.html#execution-plan after https://postgrest--4018.org.readthedocs.build/en/4018/references/observability.html#server-timing-header. 

This is more important than Execution plan (which is not that well positioned)